tests/simulations/ringtest/create_data.py

```diff
@@ -190,6 +194,36 @@ def make_ringtest_edges():
     }
     make_edges(filename="edges_AB.h5", edges=edges, wanted_attributes=wanted_attributes)
 
+    edges = Edges("RingC", "RingC", "electrical", [(0, 2), (2, 0)])
```

Comment:
maybe a small comment here to explain what we want

Reply:
Cool that we can make gap junctions in ring test circuit. In our usage, we make a single circuit for all tests? I wonder if we should make multiple circuits with limited features activated? Or maybe that adds too much overhead?

Reply:
I made a new node population C which contains C->C chemical and electrical edges. Population`C` is not included in the default `circuit_config.json`, but in `circuit_config_gj.json` to be used for the gap junction test.  
But `circuit_config_gj.json` also contains population A, B used for other ring tests, shall I change it to only read `C` for gap junction tests?

Reply:
> maybe a small comment here to explain what we want

I don't think we need a comment here. The edges of A,B don't have comments either.

Reply:
> That sounds like a good idea. We can make a habit of including only those entities that are relevant for the active test

 I will make another commit to change that